### PR TITLE
Dunham's Sports

### LINKED ISSUE #121: Dunham's Sports
http://www.dunhamssports.com/store-map/

### PR DESCRIPTION
**Pull request by AnthoneyKalasho**
Fixes #121

### CODE REVIEW

**Comment by AnthoneyKalasho** [comment]
There are two cases in the data where opening hours are listed as 12pm when in reality it should be 12am, is this case something I should fix or should I simply scrape it as it is listed.

**Comment by iandees** [comment]
Keep it as is

File: locations/spiders/dunhams_sports.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+        hoursMonString = self.store_hours(match.xpath('.//@hours_mon').extract_first().strip())
+        hoursSatString = self.store_hours(match.xpath('.//@hours_sat').extract_first().strip())
+        hoursSunString = self.store_hours(match.xpath('.//@hours_sun').extract_first().strip())
+        allHours = "Mo-Fr " + hoursMonString +"; "+ "Sa " + hoursSatString + "; " + "Su "+ hoursSunString
+
+        yield GeojsonPointItem(
+          ref=refString,
+          lat=match.xpath('.//@lat').extract_first().strip(),
```
Comment (iandees) [inline locations/spiders/dunhams_sports.py]: Cast these strings to `float` please.

File: locations/spiders/dunhams_sports.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          lon=match.xpath('.//@lng').extract_first().strip(),
```
Comment (iandees) [inline locations/spiders/dunhams_sports.py]: Case these strings to `float` please.

File: locations/spiders/dunhams_sports.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+          closeM = "00"
+        closeH = str(int(closeH) + 12)
+        closeHours = closeH + ":" + closeM
+
+        return openHours + "-" + closeHours
+      
+    def parse(self, response):
+      for match in response.xpath("//markers/marker"):
+        fullAddress=match.xpath('.//@address').extract_first().replace('<br>', ', ')\
+        #cityString = fullAddress.split(",")[1].strip()
```
Comment (iandees) [inline locations/spiders/dunhams_sports.py]: Remove commented out debug code.

File: locations/spiders/dunhams_sports.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+class DunhamsSportsSpiders(scrapy.Spider):
+    name = "dunhams_sports"
+    allowed_domains = ["http://www.dunhamssports.com"]
+    start_urls = (
+        'http://www.dunhamssports.com/wp-admin/admin-ajax.php?action=simloc_get_locations&lat=32.7258378&lng=-116.95580669999998&radius=200000',
+    )
+
+    def store_hours(self, hours):
+      openHours = hours.split("-")[0].strip()
```
Comment (iandees) [inline locations/spiders/dunhams_sports.py]: Please use 4 spaces for indent instead of 2 spaces throughout this file.

**Comment by AnthoneyKalasho** [comment]
I'll get right to it

**Comment by iandees** [comment]
Looks good, thanks!

**Comment by AnthoneyKalasho** [comment]
@iandees Unrelated but do you have any idea on how long this project will last?

**Comment by iandees** [comment]
@AnthoneyKalasho a couple weeks. I'd send an email to @nvkelso for more info.

**Pull request merged**